sdk/core/perf/inc/azure/perf/argagg.hpp

```diff
@@ -771,7 +771,7 @@ try
 {
   return this->options.at(name);
 }
-catch (const std::out_of_range const& e)
+catch (std::out_of_range const& e)
```

Comment:
@vhvb1989 how did this get checked-in past the CI?

Reply:
It got introduced here: https://github.com/Azure/azure-sdk-for-cpp/pull/2361
This should have failed compilation in CI, no?

Reply:
CI is not currently building the Performance FW.  I haven't officially added it to CI..  so, only when I try to build it locally I notice about this.

Since Perf is not officially part of Core package, I don't know if Core should build it.  And since it is not an official package, I don't know if I should create a new pipeline. I need to ask @danieljurek  what's the best option and what other languages are doing.

So, I have been checking this library out of CI by myself

Reply:
Yep, can you please file an issue. Let's get a plan together to stay green and avoid having a dev spend the time, like yourself, always having to re-build locally to verify. For most (if not all) checked-in code, verifying it is building is good to incorporate in CI.

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/2418